src/test/groovy/org/mockito/release/internal/gradle/ContributorsPluginTest.groovy

```diff
@@ -5,6 +5,11 @@ import testutil.PluginSpecification
 class ContributorsPluginTest extends PluginSpecification {
 
     def "applies"() {
+        given:
+        def conf = applyReleaseConfiguration()
```

Comment:
This seems like a logic that we could push out to setup() method of PluginSpecification, no? Repo and token are not relevant for this plugin so removing that setting from test makes the tests easier to read / maintain.